internal/generator/vector/output/splunk/splunk_test.go

```diff
@@ -75,6 +75,17 @@ compression = "none"
 default_token = "` + hecToken + `"
 [sinks.splunk_hec.encoding]
 codec = "json"
+`
+		splunkSinkTimestampKey = splunkDedot + `
+[sinks.splunk_hec]
+type = "splunk_hec_logs"
+inputs = ["splunk_hec_dedot"]
+endpoint = "https://splunk-web:8088/endpoint"
+compression = "none"
+default_token = "` + hecToken + `"
+timestamp_key = "@timestamp"
```

Comment:
In what cases do we need to override this value to something other then '@timestamp'.  Why can't we just set it to this value for all cases and not expand the API?